src/components/dialog/dialog.svelte

```diff
@@ -265,7 +265,7 @@
     gap: var(--leo-spacing-l);
   }
 
-  @media (min-width: 375px) and (min-height: 375px) {
+  @media (min-width: 480px) and (min-height: 375px) {
```

Comment:
I think this is to work in landscape/portrait on mobile
```suggestion
  @media (min-width: 480px) and (min-height: 480px) {
```